ExampleCodes/EB/STLtest/inputs

```diff
@@ -1,6 +1,13 @@
-prob_lo=  0   0  0
-prob_hi=  220 35 60
-ncells =  128 32 64
-stl_file=airfoil.stl
-max_grid_size=8
-outside_point=250.0 250.0 250.0
+eb2.geom_type             = stl
+eb2.stl_file              = airfoil.stl
+
+eb2.stl_scale             = 1                   # default is 1
+eb2.stl_center            = 0      0      0     # default is (0,0,0)
+eb2.stl_reverse_normal    = 1                   # default is 0. use "1" to see the volume of the stl object 
+
+eb2.prob_lo               = 0      0      0
+eb2.prob_hi               = 220    35     60
+eb2.ncells                = 128    32     64
+
+eb2.max_grid_size         = 8
+eb2.outside_point         = 250.0  250.0  250.0
```

Comment:
`outside_point` is no longer needed.

Reply:
I kept it because it gave me an error when I ran it:
`AMReX (7dc2081a91e1) initialized`
`ParmParse::sgetarr ParmParse::sgetarr(): eb2.outside_point not found in table`

Reply:
Right, a few lines in main.cpp related to this also need to be removed.

Reply:
oh okay I thought you meant just delete in the input file. makes sense now.